modules/data.atmosphere/R/ERA5_met_process.R

```diff
@@ -11,16 +11,11 @@
 #' 
 #' @author Dongchen Zhang
 #' @importFrom dplyr %>%
-#'
+#' @importFrom foreach %dopar%
 ERA5_met_process <- function(settings, in.path, out.path, write.db=FALSE, write = TRUE){
```

Comment:
Exposing parallel settings that assumed all cores below
```suggestion
ERA5_met_process <- function(settings, in.path, out.path, write.db=FALSE, write = TRUE, n_cores = 1){
```

Reply:
Fixed.